src/kirby/components/page/page.component.ts

```diff
@@ -105,10 +105,6 @@ export class PageComponent implements OnInit, OnDestroy, AfterViewInit, AfterCon
   @ViewChild(IonContent, { static: true }) private content: IonContent;
   @ViewChild('pageTitle', { static: false, read: ElementRef })
   private pageTitle: ElementRef;
-  @ViewChild('stickyToolbarButtons', { static: false, read: ElementRef })
-  private stickyToolbarButtons: ElementRef;
-  @ViewChild('fixedToolbarButtons', { static: false, read: ElementRef })
-  private fixedToolbarButtons: ElementRef;
   @ViewChild('simpleTitleTemplate', { static: true, read: TemplateRef })
```

Comment:
Just cleaned up the unused. Don't worry :)

Reply:
I _do_ get worried... Boyscouting should be done in relation to something you're already fixing/improving/cleaning up, not something completely unrelated... Please revert and keep focus on the thing you're trying to fix 👍